pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/scaleup-scaledown/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layout.pug
+navigationTitle: Scale Up or Down a Cluster Node
+title: Scale Up or Down a Cluster Node
+menuWeight: 84
+excerpt: Procedure to scale-up or scale-down nodes in an existing DKP cluster.
+beta: false
+enterprise: false
+---
+
+## Prerequisites
+
+- You must have the bootstrap node running with the SSH key/secrets created.
+- The export values in the environment variables section should contain the addresses of the nodes that you need to add in the section [define infrastructure][define-infrastructure].
+- Update the preprovisioned_inventory.yaml with the new host addresses.
+- Run the kubectl apply command.
+
+## Scale up a Cluster Node
+
+Follow these steps:
+
+1.  Fetch the existing preprovisioned_inventory:
+
+    ```bash
+    $ kubectl get preprovisionedinventory
```

Comment:
Remove all instances of $ at the beginning of kubectl commands because this will cause the command to fail if they use the copy button. otherwise they'd have to manually highlight everything but the $ and copy and paste it.